index.html

```diff
@@ -7544,15 +7543,16 @@ <h2 id="ConvolverNode">
               this attribute is null.
             </p>
             <p>
-              When the <code>buffer</code> has a single channel, the
-              <a>ConvolverNode</a> treats it as if it had two channels with
-              both channels containing exactly the same data. This is needed to
-              support a stereo input.
+              When the <code>buffer</code> has a single channel and there are
```

Comment:
Perhaps add a new paragraph (or sentences) saying that mono input and mono response produces a mono output and for all other cases the output is stereo.  (As a companion to the new diagram.)